index.html

```diff
@@ -65,15 +65,194 @@ <h1 class="intro-heading">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+				</div>
```

Comment:
- Hi @lincoln1883 :wave:, I am glad you have made great progress with your designs but according to the previous reviewer, he suggested that you reduce the `project image`, not to remove it. At the moment, I can't see them again. Would you please look into this?

![image](https://user-images.githubusercontent.com/99715640/227682365-e0aae470-3ad9-4e01-8aad-cf258825c764.png)

- [ ] The project card is expected to have a `width` of `327px` and `height` of `474px`.
- [ ] Another observation is that you seem to be missing some button styling according to the pallette the reviewer provided you in the previous review :wink: